car_control.py

```diff
@@ -1,137 +1,128 @@
-#!/usr/bin/env python3.7
+#!/usr/bin/env python3
 print("Loading Python modules for car_control. Please be patient.")
-import rcpy, datetime, time, math
+import rcpy, datetime, time, math, socket, json
 #import pygame
-from rcpy.servo import servo1
-from rcpy.servo import servo3
+# from rcpy.servo import servo1
+# from rcpy.servo import servo3
 #from rcpy.button import modeAI, pauseAI
 from rcpy.button import mode, pause
 from rcpy import button
 from rcpy.led import red
 from rcpy.led import green
 print("Done importing modules for now!")
 
-# This car_control routine was originally part of the OpenMV project.
-# Copyright (c) 2013-2017 Ibrahim Abdelkader <iabdalkader@openmv.io> & Kwabena W. Agyeman <kwagyeman@openmv.io>
-# This work is licensed under the MIT license, see the file LICENSE for details.
```

Comment:
Please don't strip copyright or license information.

Reply:
This was an accident, we will make sure to correct this.